lib/src/main/java/com/auth0/android/lock/views/SocialButton.java

```diff
@@ -50,13 +50,13 @@ public void onFocusChange(View v, boolean hasFocus) {
     }
 
     private StateListDrawable getTouchFeedbackBackground(@ColorInt int pressedColor, @ViewUtils.Corners int corners) {
-        final ShapeDrawable normalBackground;
+        final Drawable normalBackground;
         final ShapeDrawable pressedBackground;
 
         boolean shouldDrawOutline = pressedColor == Color.WHITE;
         if (shouldDrawOutline) {
             int outlineColor = getResources().getColor(R.color.com_auth0_lock_social_light_background_focused);
-            normalBackground = ViewUtils.getRoundedOutlineBackground(getResources(), outlineColor);
+            normalBackground = ViewUtils.getOpaqueRoundedOutlineBackground(getResources(), pressedColor, outlineColor);
```

Comment:
@lbalmaceda I left this the same because the auth config background color is being passed to `getTouchFeedbackBackground` which is then used for this new method